vulkaninfo/vulkaninfo.c

```diff
@@ -2549,6 +2551,19 @@ static void AppGpuDumpProps(const struct AppGpu *gpu, FILE *out) {
                     printf("\tmaxMultiviewViewCount     = %u\n", multiview_props->maxMultiviewViewCount    );
                     printf("\tmaxMultiviewInstanceIndex = %u\n", multiview_props->maxMultiviewInstanceIndex);
                 }
+            } else if (structure->sType == VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_MAINTENANCE_3_PROPERTIES_KHR) {
+                VkPhysicalDeviceMaintenance3PropertiesKHR *maintenance3_props = (VkPhysicalDeviceMaintenance3PropertiesKHR*)structure;
+                if (html_output) {
+                    fprintf(out, "\n\t\t\t\t\t<details><summary>VkPhysicalDeviceMaintenance3Properties</summary>\n");
+                    fprintf(out, "\t\t\t\t\t\t<details><summary>maxPerSetDescriptors    = <div class='val'>%u</div></summary></details>\n", maintenance3_props->maxPerSetDescriptors);
+                    fprintf(out, "\t\t\t\t\t\t<details><summary>maxMemoryAllocationSize = <div class='val'>%u</div></summary></details>\n", maintenance3_props->maxMemoryAllocationSize);
+                    fprintf(out, "\t\t\t\t\t</details>\n");
+                } else if (human_readable_output) {
+                    printf("\nVkPhysicalDeviceMaintenance3Properties:\n");
+                    printf("=======================================\n");
+                    printf("\tmaxPerSetDescriptors    = %u\n", maintenance3_props->maxPerSetDescriptors   );
+                    printf("\tmaxMemoryAllocationSize = %u\n", maintenance3_props->maxMemoryAllocationSize);
```

Comment:
I've recently learned that using `...<div class='val'>" PRIxLEAST64 "</div></summary>...` for types like VkDeviceSize (such as maxMemoryAllocationSize in this case) is more portable than using %llu. There are examples of this throughout vulkaninfo.